refactor: demo account to oak components - AI-460

JBR90 opened the pull request:
## Description

- [This PR](https://github.com/oaknational/oak-ai-lesson-assistant/pull/204) with mobile bugs fixed
- Some requested design changes:

- Demo user modal centered
- fonts and color font aligned with OAK



## Issue(s)

Fixes #

## How to test

1. Go to https://oak-ai-lesson-assistant-c77tv9szu.vercel-preview.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Does this PR update a package with a breaking change

codeincontext commented on apps/nextjs/src/components/AppComponents/Chat/chat-right-hand-side-lesson.tsx:
```diff
@@ -77,7 +77,7 @@ const ChatRightHandSideLesson = ({
             closeMobileLessonPullOut();
           }}
-          className="flex items-center justify-center gap-3"
+          className="mt-22 flex items-center justify-center gap-3 "
```
Should this always be 22 or does it depend on the demo status?

JBR90 replied:
Yes should be, will fix this

codeincontext reviewed (approved):
Code approved, but let's get Mike's test before merging 👍

mikeritson-oak reviewed (changes requested):
I've added some notes in the Notion ticket for feedback.

JBR90 commented:
> I've added some notes in the Notion ticket for feedback.



> I've added some notes in the Notion ticket for feedback.

Thanks Mike, I added a couple of comments in the ticket.

oak-machine-user commented:
:tada: This PR is included in version 1.12.0 :tada:

The release is available on [GitHub release](https://github.com/oaknational/oak-ai-lesson-assistant/releases/tag/v1.12.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
